<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1348</width>
    <height>788</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>-40</y>
      <width>1221</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="tabChangesFocus">
          <bool>false</bool>
         </property>
         <property name="tabStopDistance">
          <double>40.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="undo">
           <property name="text">
            <string>Undo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="save">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="redo">
           <property name="text">
            <string>Redo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit_2">
         <property name="tabStopDistance">
          <double>40.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="importButton">
         <property name="text">
          <string> Import</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="beautify">
         <property name="text">
          <string>   Beautify</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="json">
         <property name="text">
          <string> Json</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fix">
         <property name="text">
          <string> Fix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="compress">
         <property name="text">
          <string> Compress</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="decompress">
         <property name="text">
          <string> Decompress</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="minify">
         <property name="text">
          <string> Minify</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="check">
         <property name="text">
          <string> Check</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
